frontend/app/src/components/settings/accounting/AccountingTradeSettings.vue

```diff
@@ -8,13 +8,48 @@ const { t } = useI18n();
       {{ t('accounting_settings.trade.title') }}
     </template>
 
-    <CryptoToCryptoTradeSetting />
-    <GasCostSetting />
-    <TaxFreeSetting />
-    <AccountAssetMovementsSetting />
-    <CalculatePastCostBasisSetting />
-    <EthStakingTaxableAfterWithdrawalSetting />
-    <CostBasisFeesSetting />
-    <CostBasisMethodSetting />
+    <SettingsItem>
+      <template #title>
+        {{ t('accounting_settings.trade.titles.crypto2crypto') }}
+      </template>
+      <CryptoToCryptoTradeSetting />
+    </SettingsItem>
```

Comment:
Please move this inside `CryptoToCryptoTradeSetting` instead.
`CryptoToCryptoTradeSetting` already has `SettingsOption, you don't need to use `SettingsItem` anymore.
Use `SettingsItem` if there are multiple settings.

Also check all the usages below too